### DISCUSSION TITLE
3rd Party Authorization

### DISCUSSION THREAD

**Discussion by neverm8** [community/community#29224] [Codespaces]
Hi all, first post so gidday!

What's the score with this - don't allow or are they sweet?

What's the main risk if you allow them access in short terms?

I just downloaded 'CodeStream' so hopefully all good lol

Cheers

**Comment by tuves** [comment]
Hi @neverm8, 

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your duplicate post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have deleted the other one, and we ask that you no longer cross-post.
For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
